internal/storage/v2/api/tracestore/reader.go

```diff
@@ -61,7 +61,7 @@ type Reader interface {
 	// of matching trace IDs. This is useful in some contexts, such as batch jobs, where a
 	// large list of trace IDs may be queried first and then the full traces are loaded
 	// in batches.
-	FindTraceIDs(ctx context.Context, query TraceQueryParams) iter.Seq2[[]pcommon.TraceID, error]
+	FindTraceIDs(ctx context.Context, query TraceQueryParams) iter.Seq2[[]GetTraceParams, error]
```

Comment:
@yurishkuro How do you feel about this? I was trying to re-use the same type but it does feel a little awkward to have a function return a type with `Params` in the name.

Reply:
Yes , I would create a different return type, even if it's structurally the same.